src/css/header.css

```diff
@@ -1,3 +1,119 @@
+/** ========= HEADER ========= */
+.page-header {
+}
+
+.header-container {
+  position: absolute;
+  width: 335px;
+  top: 20px;
+  left: 50%;
+  transform: translate(-50%);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border: 1px solid #fbfbfb;
+  border-radius: 8px;
+  background: #fbfbfb;
+}
+
+.header-nav {
+}
+
+.header-logo {
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 14px;
+  letter-spacing: -0.03em;
+  color: #1b414b;
+  display: block;
+  padding: 12px 0 13px 20px;
+}
+
+.header-nav-list,
+.header-soc {
+  display: none;
+}
+
+.header-nav-item {
+}
+.header-nav-link {
+}
+.current {
+}
+.active {
+}
+.header-soc {
+}
+.header-soc-item {
+}
+.header-soc-link {
+}
+.header-soc-icon {
+}
+
+.burger-btn {
+  padding: 0;
+  border: none;
+  background-color: transparent;
+}
+
+.burger-icon {
+  fill: #1b414b;
```

Comment:
this icon is drawn using strokes. You need to write styles for this icon similar to those used by our lecturer.